manageability-docker/vision-docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM jrei/systemd-ubuntu:20.04 as base
```

Comment:
@gblewis1 - Do you think we should have the copyright added to all of these Docker related files?  And scripts?

Reply:
@nmgaston We could do that.. maybe SPDX identifier would be best.

Reply:
I talked to Natalie and Yeng Liong and we'll do this in another PR.